site/reference/pkg/cat/README.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Cat"
+linkTitle: "cat"
+type: docs
+description: >
+  Print the resources in a package
+---
+
+<!--mdtogo:Short
+    Print the resources in a package
+-->
+
+{{< asciinema key="pkg-cat" rows="10" preload="1" >}}
+
+Cat prints the resources in a package as yaml to stdout.
+
+Cat is useful for printing only the resources in a package which might
+contain other non-resource files.
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+```
+kpt cfg cat [DIR]
+
+DIR:
+  Path to a package directory. Defaults to the current working directory.
+```
+
+<!--mdtogo-->
+
+### Examples
+
+<!--mdtogo:Examples-->
+
+```sh
+# print Resource config from current directory
+kpt cfg cat
+```
+
+<!--mdtogo-->
+
+#### Flags
+
+```sh
+--annotate
```

Comment:
what does this do? annotate path/index/package annotations? Can you rationalize the use cases for all the flags here?

Reply:
`annotate path/index/package annotations?` => Yes.

Most of the flags try to replicate what `kpt fn source/sink` does. Removed such flags.

Reply:
If source/sink is used instead, what's the use case for keeping annotate flag?

Reply:
This will be useful for users to see the file origins.